tests/test_dse.py

```diff
@@ -322,7 +362,46 @@ def test_contracted_alias_shape_after_blocking():
     # Leads to 2D aliases
     eqn = Eq(u.forward, ((u[t, x, y, z] + u[t, x, y+1, z+1])*3*f +
                          (u[t, x, y+2, z+2] + u[t, x, y+3, z+3])*3*f + 1))
-    op0 = Operator(eqn, dse='noop', dle=('advanced', {'openmp': True}))
+    op0 = Operator(eqn, dse='basic', dle=('advanced', {'openmp': True}))
+    op1 = Operator(eqn, dse='aggressive', dle=('advanced', {'openmp': True}))
+
+    y0_blk_size = op1.parameters[9]
+    z_size = op1.parameters[-1]
```

Comment:
isn't it fragile ?

Reply:
This is copied from the already existing test. This will be probably removed however.